aquadoggo/src/db/stores/test_utils.rs

```diff
@@ -107,80 +111,100 @@ pub fn test_delete_operation(previous_operations: Vec<OperationId>) -> Operation
     .unwrap()
 }
 
-pub async fn test_db(no_of_entries: usize, with_delete: bool) -> SqlStorage {
+pub fn test_key_pairs(no_of_authors: usize) -> Vec<KeyPair> {
+    let mut key_pairs = vec![KeyPair::from_private_key_str(DEFAULT_PRIVATE_KEY).unwrap()];
+
+    for index in 1..no_of_authors {
+        key_pairs.push(KeyPair::new())
+    }
+
+    key_pairs
+}
+
+pub async fn test_db(
```

Comment:
Yeh, I posted a note somewhere saying I should be trying to use that, I definitely went to spend some time figuring it out / reading Piets tests. I was gunna try, but then I thought that in these tests we actually want to use the db in most cases, except maybe when testing the default trait implementations which only combine other db operations. So then I didn't look into it much more. If you have some suggestions, I'm up for looking into it.

Reply:
another slightly off-topic comment: I'm sad I couldn't get the test utils exporting properly from `p2panda-rs` before doing all this work in `aquadoggo`..... some of those fixtures would have been useful, and it would be really nice to have a consistent testing pattern across both repos. I know how to do it now, but there is also a lot of refactoring to be done, inclusing using the storage provider in our mock node, which was dependant on these changes, so it all had to wait.

Reply:
> .. but then I thought that in these tests we actually want to use the db in most cases

Yeah, that makes sense! Let's keep it like that!

> I know how to do it now, but there is also a lot of refactoring to be done

A ticket for the future will be enough for now, I think the testing looks alright.